src/game/GameScreen.js

```diff
@@ -28,13 +30,19 @@ export default class GameScreen extends React.Component {
         const idGame = navigation.state.params.idGame
             ? navigation.state.params.idGame
             : '(...)';
-        const numberOfPlayers = 2;
+        let numberOfPlayers = '#';
+        if (navigation.state.params.soloGame !== undefined) {
+            numberOfPlayers =
+                navigation.state.params.soloGame == true ? '1' : '2';
```

Comment:
!!navigation.state.params.soloGame ? '1' : '2'

Why not use an int instead of string ?

Reply:
I choose this besause it's for dysplay only, an I dont like that a variable can be alteratively a string ('#') and then a number (1 or 2)